src/pfs/pfs.proto

```diff
@@ -200,6 +200,17 @@ message DeleteRepoRequest {
   bool force = 2;
 }
 
+// DeleteReposRequest is used to delete more than one repo at once.
+message DeleteReposRequest {
```

Comment:
this might be obvious, but why not call this DeleteProjects?

Reply:
Because it deletes repos, not projects☺